### ISSUE TITLE
Zuzalu Grantees Having Issues with Saving Edits in Builder 

### ISSUE THREAD

**Issue by M0nkeyFl0wer** [gitcoinco/grants-stack#2834]
### What was the URL where the error occurred?

https://builder.gitcoin.co/#/chains/1/registry/0x03506eD3f57892C85DB20C36846e9c808aFe9ef4/projects/405

### What happened?

Two different grantees in the telegram thread for the Zuzalu round reported having issues when trying to edit their proposal in Builder before submitting it on Explorer. Both showed similar screenshots of the error message.

 
![photo_2024-01-24_20-58-09](https://github.com/gitcoinco/grants-stack/assets/91565301/6f8b8cb3-5537-4d0c-b4ba-14c9a9ac7239)

Upon further discussion with one of the grantees they flagged other concerns with their experience and shared this screen recording: https://share.descript.com/view/Z89wAxF19AO

Also flagged in this video shared by the user were two other issues when editing their proposal. First new issue was an errror message  in the field for the website url which seems to be falsely triggered when the url was correct and could be fixed by a quick delete and retyping of the same letter in the website field to reset it. The other issues was ux issue while trying to verifying github account information.  It seems it wasn't clear to them the flow they needed to take to verify their organization. This may not have been  

### What should have happened instead?

It appears that the users were trying to make changes their proposals in Builder but when they went to save the changes they made the signature transaction did not go through and they were unable to save their proposal. Ideally they would be able to edit their proposal. 

### Steps to Reproduce

I have not had a chance to try to reproduce this error message. 

The steps involved in reproducing would be to visit Builder and to go through the steps to create a new proposal and save it. Then log out, return to the page, log back in with your wallet, make changes and see if they will save. 

To be clear this is unrelated to grantees editing after applying to the round. Both the grantees that reached out are trying to make changes and save them before applying to the round. 

### What chain(s) did the error occur on?

Optimism

**Comment by M0nkeyFl0wer** [comment]
FYI the same user shared a screenshot of a log they saw when getting the error.

![photo_2024-01-24_18-12-57 (2)](https://github.com/gitcoinco/grants-stack/assets/91565301/3ff494b3-0ab4-414e-8b86-23bf0add210e)

![image](https://github.com/gitcoinco/grants-stack/assets/91565301/05981006-2036-4f88-b097-dc8c512fea85)

For what it's worth this reminds me of what we saw when we were having issues with payout contracts at the end of the gg19 and we got that same error message very quickly after trying to initiate a signature. Not sure if there would be any connection but seemed worth mentioning.
